lib/system.nim

```diff
@@ -4203,3 +4213,22 @@ when not defined(js):
 type
   ForLoopStmt* {.compilerProc.} = object ## special type that marks a macro
                                          ## as a `for-loop macro`:idx:
+
+when defined(genode):
```

Comment:
I don't see the need for these to be in ``system.nim`` (but I will accept it). Why not introduce a ``std / genode`` module?

Reply:
It would be difficult to decide what to put in the standard library and what to leave out, so I would rather put the bare minimum in system and the rest in Nimble, where versions can be pegged to Genode releases. Otherwise things like audio clients and server or driver frameworks could end up here.